tests/index.js

```diff
@@ -39,7 +39,9 @@ describe('configurations', function() {
       assert.equal(plugin.configs.all.rules[configName], 2);
     });
     rules.forEach(function(ruleName) {
-      assert(plugin.configs.all.rules['react/' + ruleName]);
+      if (!plugin.deprecatedRules[ruleName]) {
```

Comment:
we should assert instead that every rule is either in `deprecatedRules` or `all`, rather than just skipping the deprecated ones.